### PR TITLE
Subscription Update for Wink

### PR DESCRIPTION
**Pull request by danielrischar**
Updates subscription methods for wink devices in preparation for hub controller changes.

Also fixes some RAML and translator typos.

### CODE REVIEW

File: opent2t.p.hub/com.wink.hub/js/thingTranslator.js [hunk 1]
```diff
@@ -9,6 +9,55 @@ var request = require('request-promise');
...
+        // Additionally, do not return devices that have been marked as hidden by Wink (hidden_at is a number)
+        // This state is used by third party devices (such as a Nest Thermostat) that were connected to a
+        // Wink account and then removed.  Wink keeps the connection, but marks them as hidden.
+        if ((winkDevice.model_name !== 'HUB')  && 
+            typeof opent2tInfo !== 'undefined' &&
+            (winkDevice.hidden_at == undefined || winkDevice.hidden_at == null))
+        {
+            // set the opent2t info for the wink device
+            var deviceInfo = {};
+            deviceInfo.id = this._getDeviceId(winkDevice);
```
Comment (arjun-msft) [inline opent2t.p.hub/com.wink.hub/js/thingTranslator.js]: >id [](start = 23, length = 2)

needs to be controlId syntax

**Comment by arjun-msft** [comment]
> /* jshint esversion: 6 */

this file looks all wonky, are you sure you're synced right?

---
Refers to: opent2t.p.hub/com.wink.hub/js/thingTranslator.js:1 in a530e38. [](commit_id = a530e3838df617033b458039500fd125cfc49f4d, deletion_comment = False)

**Comment by arjun-msft** [comment]
:clock1:

**Comment by arjun-msft** [comment]
> /* jshint esversion: 6 */

Outside this file, everything else looks good

---
In reply to: [262046357](https://github.com/openT2T/translators/pull/125#issuecomment-262046357) [](ancestors = 262046357)

---
Refers to: opent2t.p.hub/com.wink.hub/js/thingTranslator.js:1 in a530e38. [](commit_id = a530e3838df617033b458039500fd125cfc49f4d, deletion_comment = False)

**Comment by danielrischar** [comment]
> /* jshint esversion: 6 */

Looks like it's an issue with CodeFlow.  I pulled the code that produces the list of platforms out into a separate function, and it's having a hard time deciding where it should go.

---
In reply to: [262046357](https://github.com/openT2T/translators/pull/125#issuecomment-262046357) [](ancestors = 262046357)

---
Refers to: opent2t.p.hub/com.wink.hub/js/thingTranslator.js:1 in a530e38. [](commit_id = a530e3838df617033b458039500fd125cfc49f4d, deletion_comment = False)

**Pull request merged**